configure.ac

```diff
@@ -74,14 +74,11 @@ AS_IF([test "x$enable_gtk3" = xyes],
 AM_CONDITIONAL([GTK3], [test "x$gtk_package" = "xgtk+-3.0"])
 
 # GTK/GLib/GIO checks
-gtk_modules="$gtk_package >= $gtk_min_version glib-2.0 >= 2.28"
-gtk_modules_private="gio-2.0 >= 2.28 gmodule-no-export-2.0"
+gtk_modules="$gtk_package >= $gtk_min_version glib-2.0 >= 2.32"
+gtk_modules_private="gio-2.0 >= 2.32 gmodule-no-export-2.0"
```

Comment:
This *might* work (overwriting with newer GLib):
```diff
diff --git a/scripts/cross-build-mingw.sh b/scripts/cross-build-mingw.sh
index f3cf9b5..6aab88a 100755
--- a/scripts/cross-build-mingw.sh
+++ b/scripts/cross-build-mingw.sh
@@ -52,7 +52,7 @@ fetch_and_unzip()
 {
   local basename=${1##*/}
   curl -L -# "$1" > "$basename"
-  unzip -q "$basename" -d "$2"
+  unzip -qo "$basename" -d "./$2"
   rm -f "$basename"
 }
 
@@ -86,6 +86,11 @@ cd "$BUILDDIR"
 
 mkdir _deps
 fetch_and_unzip "$BUNDLE_ZIP" _deps
+if [ "$GTK3" = no ]; then
+  # get newer GLib
+  fetch_and_unzip "http://win32builder.gnome.org/packages/3.6/glib-dev_2.34.3-1_win32.zip" _deps
+  fetch_and_unzip "http://win32builder.gnome.org/packages/3.6/glib_2.34.3-1_win32.zip" _deps
+fi
 # fixup the prefix= in the pkg-config files
 sed -i "s%^\(prefix=\).*$%\1$PWD/_deps%" _deps/lib/pkgconfig/*.pc
 
```

Alternatively, this is trickier but might also work (add files from the GTK2 bundle onto the GTK3 one, not overwriting):
```diff
diff --git a/scripts/cross-build-mingw.sh b/scripts/cross-build-mingw.sh
index f3cf9b5..6f4b2c7 100755
--- a/scripts/cross-build-mingw.sh
+++ b/scripts/cross-build-mingw.sh
@@ -52,7 +52,7 @@ fetch_and_unzip()
 {
   local basename=${1##*/}
   curl -L -# "$1" > "$basename"
-  unzip -q "$basename" -d "$2"
+  unzip -qn "$basename" -d "./$2"
   rm -f "$basename"
 }
 
@@ -85,7 +85,8 @@ mkdir "$BUILDDIR"
 cd "$BUILDDIR"
 
 mkdir _deps
-fetch_and_unzip "$BUNDLE_ZIP" _deps
+fetch_and_unzip "$GTK3_BUNDLE_ZIP" _deps
+[ "$GTK3" = yes ] || fetch_and_unzip "$BUNDLE_ZIP" _deps
 # fixup the prefix= in the pkg-config files
 sed -i "s%^\(prefix=\).*$%\1$PWD/_deps%" _deps/lib/pkgconfig/*.pc
 
```